### DISCUSSION TITLE
FreePBX docker container sounds awesome.

### DISCUSSION THREAD

**Discussion by penguinjeff** [community/community#45279] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Looking to convert my FreePBX from a vm to a docker container.
I see https://hub.docker.com/r/tiredofit/freepbx By [tiredofit](https://hub.docker.com/u/tiredofit) • Updated 9 months ago
When I try to look at the docker-compose.yml
https://hub.docker.com/r/tiredofit/examples/docker-compose.yml
I get
404

I'd really like to get this working. My vm took a bad turn and I will need to rebuild it. vm's take a bit more memory as well.
I'll first look at starting a mariadb container and how to properly do that then I'll look at some other
docker-compose.yml files I have for pihole and unifi I may be able to figure this out on my own but it would be nice to see the example for freepbx docker container.

The instability could be because I was using the newest one not the stable version.

**Comment by airtower-luna** [comment, marked as answer]
It looks like you're looking for help with a software hosted on GitHub, maybe https://github.com/tiredofit/docker-freepbx? In that case you should look for support information *in that repository*, usually in the README or a file called SUPPORT. If those don't say anything, opening an issue or discussion there is usually good. :slightly_smiling_face:
